.github/workflows/main-build.yml

```
@@ -5,8 +5,15 @@ on:
     branches: 
       - main 
       - release/*
+    paths-ignore:
+      - '.github/**'
```

```suggestion
      - '.github/**'
      - '!.github/workflows/main-build.yml'
```
You do not want to ignore all of `.github`. This should also be ran if the workflow file changed.